tests/tests.js

```diff
@@ -88,4 +88,35 @@ QUnit.test('processTree: single node', function(assert) {
   serializer.processTree(node, 0);
   assert.equal(Object.keys(serializer.srcHoles).length, 0);
   assert.equal(Object.keys(serializer.frameHoles).length, 0);
+});
+
+QUnit.test('processTree: no closing tag', function(assert) {
+  var serializer = new HTMLSerializer();
+  var img = document.createElement('img');
+  serializer.processTree(img);
+  assert.equal(serializer.html[0], '<img ');
+  assert.equal(
+    serializer.html[1],
+    `style="${window.getComputedStyle(img, null).cssText}" `
+  );
+  assert.equal(serializer.html[2], '>');
+  assert.equal(serializer.html.length, 3);
+  assert.equal(Object.keys(serializer.srcHoles).length, 0);
```

Comment:
I'd leave these last two lines off so this test is just about the fact that no closing img tag is present (same below).